WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -299,6 +305,23 @@ private void handleOpenPageIntent(Intent intent) {
         }
     }
 
+    private void launchZendeskMyTickets() {
+        if (isFinishing() || getIntent() == null) {
+            return;
+        }
+
+        // leave the Main activity showing the ME page, so when the user comes back from Help&Support the app is in
+        // the right section.
+        mBottomNav.setCurrentPosition(PAGE_ME);
+
+        // init selected site, this is the same as in
+        initSelectedSite();
+
+        ActivityLauncher.viewZendeskTickets(
```

Comment:
Nitpick: This is way less than 120 chars, I don't think you need to separate the parameters in different lines. Even when it's done it's generally recommended to separate each one, although we have not always been able to follow that pattern.

Reply:
thanks - addressed in 76d8cff5be